pkg/compute/init.go

```diff
@@ -185,17 +190,26 @@ func (c *InitCommand) Exec(in io.Reader, out io.Writer) (err error) {
 
 	progress.Step("Initializing package...")
 
-	if err := language.Initialize(progress); err != nil {
-		fmt.Println(err)
-		return fmt.Errorf("error initializing package: %w", err)
+	if language.Name != "other" {
+		if err := language.Initialize(progress); err != nil {
+			fmt.Println(err)
+			return fmt.Errorf("error initializing package: %w", err)
+		}
 	}
 
 	progress.Done()
 
 	text.Break(out)
 
 	text.Description(out, fmt.Sprintf("Initialized package %s to", text.Bold(m.Name)), abspath)
-	text.Description(out, "To publish the package (build and deploy), run", "fastly compute publish")
+
+	if language.Name == "other" {
+		text.Description(out, "To package a WASM binary for deployment, run", "fastly compute pack")
```

Comment:
how about "a pre-compiled WASM binary"? Concerned that this implies the build will output a wasm binary.